### PR TITLE
Talent portal

### LINKED ISSUE #993: Modify the Demo application for pruposes of CV intake
Instead of a prize draw, transform the demo into a CV collection page for job candidates at blindnet. If needed, make issues for improvement of tutorial and everything else so that anyone can build this application easily.

### PR DESCRIPTION
**Pull request by milstan**
Closes blindnet-io/product-management/issues/993

### CODE REVIEW

File: talent_portal/js/submit.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+
+    fetch('https://devkit-pce-staging.azurewebsites.net/v0/user-events/consent', {
+      method: 'POST',
+      body: JSON.stringify({
+          dataSubject: {
+            id: formData.get('email'),
+            schema: 'dsid',
+          },
+          consentId: '28b5bee0-9db8-40ec-840e-64eafbfb9ddd',
```
Comment (m4rk055) [inline talent_portal/js/submit.js]: This is the same consent from the demo app. Shouldn't we have a different one?

File: talent_portal/js/submit.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    .then(result => {
+      console.log('Success:', result);
+
+    })
+    .catch(error => {
+      console.error('Error:', error);
+    });
+
+    // submit data
+    fetch('https://blindnet-connector-demo-staging.azurewebsites.net/form', {
```
Comment (m4rk055) [inline talent_portal/js/submit.js]: Saving the data should be done only after the consent is successfully registered. Put this code inside of `then` of the previous `fetch` call.

File: talent_portal/thanx.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+<body>
+
+  <div class="container-fluid p-3 vh-100 justify-content-center">
+
+    <div class="row h-100 align-items-center justify-content-center">
+
+
+      <div class="row h-75 align-self-center align-items-center justify-content-center">
+        <div class="col-md-6 col-lg-5 text-center">
+          <h1>thanx!</h1>
```
Comment (m4rk055) [inline talent_portal/thanx.html]:
```suggestion
          <h1>thanks!</h1>
```

**Review by m4rk055 — CHANGES_REQUESTED** [review]

File: talent_portal/js/submit.js (hunk 1 above)
Comment (milstan) [reply talent_portal/js/submit.js]: Changed to a new consent

File: talent_portal/js/submit.js (hunk 2 above)
Comment (milstan) [reply talent_portal/js/submit.js]: done

**Review by m4rk055 — APPROVED** [review]

**Review by jboileau99 — APPROVED** [review]

**Pull request merged**